populate_db.py

```diff
@@ -0,0 +1,107 @@
+import pyphen, requests, json
```

Comment:
usually guidelines are one import per line, in alphabetical order

Reply:
duly noted